misk-redis/src/main/kotlin/misk/redis/Redis.kt

```diff
@@ -0,0 +1,55 @@
+package misk.redis
+
+import java.time.Duration
+
+/** A Redis client. */
+interface Redis {
+  /**
+   * Deletes a single key.
+   *
+   * @param key the key to delete
+   * @return 0 if the key was not deleted, 1 if the key was deleted
+   */
+  fun del(key: String): Long
+
+  /**
+   * Deletes multiple keys.
+   *
+   * @param keys the keys to delete
+   * @return 0 if none of the keys were deleted, otherwise a positive integer
+   *         representing the number of keys that were deleted
+   */
+  fun del(vararg keys: String): Long
+
+  /**
+   * Retrieves a key.
+   *
+   * @param key the key to retrieve
+   * @return a string value if the key was found, null if the key was not found
+   */
+  fun get(key: String): String?
```

Comment:
might be a good use case for [`operator get` and `operator set`](https://kotlinlang.org/docs/reference/operator-overloading.html#indexed)

Reply:
Ah good point, `get` and `set` can be called with the indexing operator